### PR TITLE
0.3.2

### PR DESCRIPTION
**Pull request by Shelnutt2**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Shelnutt2** [comment]
@conda-forge-admin, please rerender

**Comment by Shelnutt2** [comment]
@mariusvniekerk you recently helped with the [r-tiledb-feedstock](https://github.com/conda-forge/r-tiledb-feedstock) package in switching to `tinytest`. We've been stuck on this dependency update for a while because the windows build package is failing with `make: *** No rule to make target '$(CXX11STD)''.  Stop.`.

This update doesn't change anything with the build structure compared to the last successful release of 0.3.0. Any chance you have some insights, or can point us to someone who can help, into why on windows it can't find a c++11 compiler?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -28,16 +28,19 @@ build:
 requirements:
   build:
     - {{ posix }}zip               # [win]
+    - make
```
Comment (mariusvniekerk) [inline recipe/meta.yaml]:
```suggestion
    - make                             # [not win]
    - m2-make                          # [win]
    - {{ compiler('c') }}              # [not win]
    - {{ compiler('m2w64_c') }}        # [win]
    - {{ compiler('cxx') }}            # [not win]
    - {{ compiler('m2w64_cxx') }}      # [win]
```

And then you'll need to rerender

Comment (mariusvniekerk) [reply recipe/meta.yaml]: mostly just taken from the dplyr conda-forge recipe

Comment (Shelnutt2) [reply recipe/meta.yaml]: @mariusvniekerk thank you for the hint and suggestion!

**Comment by Shelnutt2** [comment]
@conda-forge-admin, please rerender

**Pull request merged**